dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+                        .andExpect(status().isOk())
+                        .andExpect(
+                            jsonPath("$", GroupMatcher.matchGroupEntry(submitters.getID(), submitters.getName())));
+    }
+
+    @Test
+    public void getCollectionSubmittersGroupTestCollectionAdmin() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group submitters = collectionService.createSubmitters(context, collection);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isOk())
+                        .andExpect(
+                            jsonPath("$", GroupMatcher.matchGroupEntry(submitters.getID(), submitters.getName())));
+    }
+
+    @Test
+    public void getCollectionSubmittersGroupUnAuthorizedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        collectionService.createSubmitters(context, collection);
+        context.restoreAuthSystemState();
+
+        getClient().perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void getCollectionSubmittersGroupForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        collectionService.createSubmitters(context, collection);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isForbidden());
+    }
+
+    @Test
+    public void getCollectionSubmittersGroupNoContentTest() throws Exception {
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void getCollectionSubmittersGroupWrongCollectionUuidResourceNotFoundTest() throws Exception {
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + UUID.randomUUID() + "/submittersGroup"))
+                        .andExpect(status().isNotFound());
+    }
+
+
+    @Test
+    public void postCollectionSubmitterGroupCreateSubmitterGroupSuccess() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        MetadataRest metadataRest = new MetadataRest();
+        metadataRest.put("dc.description", new MetadataValueRest("testingDescription"));
```

Comment:
again use additional metadata in one test and verify that they are stored but write most of test for the most frequent scenario where metadata are not used